internal/oauth/oauth.go

```diff
@@ -56,6 +60,15 @@ const (
 	GovClientID = "0oabtyfelbTBdoucy297" // GovClientID for production
 )
 
+// Appends user agent if run from container.
+func maybeAddUserAgentPostfix() {
+	_, runningFromContainer := os.LookupEnv("BUILD_CONTEXT")
+
+	if runningFromContainer && !strings.Contains(config.UserAgent, userAgentContainerPostfix) {
+		config.UserAgent = fmt.Sprintf("%s (%s)", config.UserAgent, userAgentContainerPostfix)
```

Comment:
Per @drinkbird comment on the JIRA ticket 

> Thanks for flagging this. By doing some further reading, I see that what comes after "product/version" is "(comment)". We can extend what we have to include the environment in all cases.

Source: https://www.rfc-editor.org/rfc/rfc9110#name-parameters

Reply:
updated